ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-segment-security.j2

```diff
@@ -0,0 +1,60 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - router segment-security #}
+{% if router_segment_security is arista.avd.defined %}
+!
+router segment-security
+{%     if router_segment_security.enabled is arista.avd.defined(true) %}
+   no shutdown
+{%     endif %}
+{%     if router_segment_security.policies is arista.avd.defined %}
+{%         for policy in router_segment_security.policies %}
+   !
+   policy {{ policy.name }}
+{%             for sequence in policy.sequence_numbers | arista.avd.natural_sort('sequence') %}
+{%                 set eos_cli = sequence.action %}
```

Comment:
Add variable check for sequence.action enclosing the full section, since without it we cannot create any CLI.